.github/workflows/tests.yaml

```diff
@@ -5,12 +5,12 @@ jobs:
     strategy:
       fail-fast: False
       matrix:
-        ubuntu: ['ubuntu-18.04', 'ubuntu-20.04']
+        ubuntu: ['ubuntu-20.04', 'ubuntu-22.04']
```

Comment:
Why not ubuntu-latest?

Sadly, there isn't the option of doing `['ubuntu-earliest', 'unbuntu-second-latest', 'ubuntu-latest']

Reply:
No particular reason - I just prefer to be explicit